plugins/calendar-resources/src/components/EventElement.svelte

```diff
@@ -16,17 +16,18 @@
   import calendar, { CalendarEventPresenter, Event } from '@hcengineering/calendar'
   import { Doc } from '@hcengineering/core'
   import { getClient } from '@hcengineering/presentation'
-  import { Component, MILLISECONDS_IN_MINUTE, showPopup, tooltip } from '@hcengineering/ui'
+  import { Component, MILLISECONDS_IN_MINUTE, deviceSizes, showPopup, tooltip } from '@hcengineering/ui'
   import view, { ObjectEditor } from '@hcengineering/view'
   import { showMenu } from '@hcengineering/view-resources'
   import { calendarStore, isVisible } from '../utils'
   import EventPresenter from './EventPresenter.svelte'
+  import { log, log } from 'console'
```

Comment:
Hi, I think we do not require this import.

Reply:
Deleted line 24